### PR TITLE
Improve `upgarde` command with additional information and spinner

### PR DESCRIPTION
**Pull request by aabedraba**
feat: add versions to upgrade command

feat: add spinner to upgrade command

fix: remove name option from install and upgarde cli

image

This PR...

## Changes

-

## Fixes

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: cmd/kusk/cmd/install.go [hunk 1]
```diff
@@ -37,15 +37,16 @@ import (
...
+	noDashboard      bool
+	noEnvoyFleet     bool
+	releaseNamespace string
 )
 
+const releaseName = "kusk-gateway"
+
 func init() {
 	rootCmd.AddCommand(installCmd)
-	installCmd.Flags().StringVar(&releaseName, "name", "kusk-gateway", "installation name")
```
Comment (kylehodgetts) [inline cmd/kusk/cmd/install.go:48]: This flag needs to remain to allow users to override the release name.

**Review by kylehodgetts — DISMISSED** [review]

File: cmd/kusk/cmd/install.go (hunk 1 above)
Comment (aabedraba) [reply cmd/kusk/cmd/install.go:48]: I removed it becuase there is a bug introduced by it: 

https://github.com/kubeshop/kusk-gateway/blob/main/cmd/kusk/cmd/install.go#L108

We are using the flag to also search for the chart, so if a user uses a custom name, the command stops working 

```sh
kusk install --name custom
⨯ Installing kusk gateway (error: process error: exit status 1
output: Release "custom" does not exist.
``` 

Should I still add it? @kylehodgetts

Comment (aabedraba) [reply cmd/kusk/cmd/install.go:48]: Also, I think `upgarde` command should not receive a releaseName, right? It should just upgrade the existing kusk-gateway distribution, I'd think. So we would have to add logic to look for the existing kusk-gateway in the cluster, get it namespace 🤔

Comment (jasmingacic) [reply cmd/kusk/cmd/install.go:48]: `releaseName` here is used to pull images. If we use it dynamically the users will end up with an error

**Review by jasmingacic — APPROVED** [review]
Looks legit

**Pull request merged**